src/themes/default/tokens/sizes.js

```diff
@@ -0,0 +1,3 @@
+const sizes = {}
```

Comment:
Same with sizes across the theme - should everything just fall back to these?

Reply:
Sizes would be used for widths/heights/etc. But I could see this going away.